src/main/java/hello/service/CustomerService.java

```diff
@@ -1,4 +1,85 @@
 package hello.service;
 
+import hello.dao.CustomerRepository;
+import hello.exceptions.MultiErrorException;
+import hello.exceptions.NotFoundException;
+import hello.model.Customer;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.springframework.http.ResponseEntity.status;
+
+@Service
 public class CustomerService {
-}
+    private CustomerRepository repository;
+
+    @Autowired
+    public CustomerService(CustomerRepository repository) { //qaulifiers say which implementaion you are using
+        this.repository = repository;
+    }
+
+
+    public Customer addCustomer(Customer customer) {
+        boolean isError = false;
+        List<String> errors = new ArrayList<>(); //make new list which will contrain of errors
+        if (StringUtils.isBlank(customer.getName())) {
+            isError = true;
+            errors.add("Customer name needs to be entered");
+        }
+
+        if (customer.isBookingConfirmed() == null) {
+            isError = true;
+            errors.add("Booking request cannot be empty");
+        }
+
+        if (isError) {
```

Comment:
Consider using `!errors.isEmpty()` here to remove the need for the `isError` variable